packages/core/admin/ee/server/tests/review-workflows.test.api.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+const { createStrapiInstance } = require('../../../../../../test/helpers/strapi');
+const { createAuthRequest, createRequest } = require('../../../../../../test/helpers/request');
+
+const edition = process.env.STRAPI_DISABLE_EE === 'true' ? 'CE' : 'EE';
+
+if (edition === 'CE') {
+  test('Review-workflows (skipped)', () => {
+    expect(true).toBeTruthy();
+  });
+  return;
+}
```

Comment:
I've just seen a nice util in @Marc-Roig PR that could apple here - https://github.com/strapi/strapi/pull/15594
`describeOnCondition`

it works like `describeOnCondition(edition === 'EE')('Review workflows', () => {...`

Reply:
Yes, actually to avoid massive conflict, I didn't implemented it in this one 👍